pages/DSNP/Operations.md

```diff
@@ -0,0 +1,39 @@
+# Operations
+
+DSNP implementations perform well-defined DSNP Operations and generate DSNP State Change Records.
+
+## Control Keys and Proofs
+
+Each invocation of a DSNP Operation MUST have verifiable approval of the acting principal(s) via a Control Key Ownership Proof.
+The precise data type and representation of both the Control Key and the Control Key Ownership Proof MUST be defined by each DSNP implementation.
+For example, an implementation might use the public key of an asymmetric key pair as a control key, and provide a proof for each operation by producing a cryptographic signature of the user's DSNP Identifier and some nonce value.
```

Comment:
Control keys also must be unique to an implementation - is that specified elsewhere?  Could we say it must be an implementation-unique public key of a cryptographic key pair of at least N bits?